lib/cb/utils/api.rb

```diff
@@ -57,10 +57,14 @@ def cb_delete(path, options = {}, &block)
       def execute_http_request(http_method, uri, path, options = {}, &block)
```

Comment:
Not sure I agree that it's larger than this PR - now is as good a time as ever for cleaning up this method. I would agree if this method wasn't being changed as part of this PR, but it is :) It's not really a time-sensitive PR either, so taking an extra hour or so to clean it up would be good in the long run as we wouldn't have to add a card to revisit it later.